tests/integ/test_udf.py

```diff
@@ -1660,20 +1660,20 @@ def add_one_df_pandas_annotation(df: pandas.DataFrame) -> pandas.Series:
             DateType,
             [[datetime.date(2021, 12, 20)]],
             ("<class 'pandas._libs.tslibs.timestamps.Timestamp'>",),
-            ("datetime64[ns]",),
+            ("datetime64[s]", "datetime64[ns]", "datetime64[us]"),
         ),
         (ArrayType, [[[1]]], ("<class 'list'>",), ("object",)),
         (
             TimeType,
             [[datetime.time(1, 1, 1)]],
             ("<class 'pandas._libs.tslibs.timedeltas.Timedelta'>",),
-            ("timedelta64[ns]",),
+            ("'timedelta64[s]", "timedelta64[ns]", "'timedelta64[us]"),
```

Comment:
Is the single quotes in this line intended?

Reply:
Oh great catch! Fixing...